### PR TITLE
Update Readme along with badges & description

### LINKED ISSUE #43: Update Readme
Add badges in the readme.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
Fix #43 . @deshraj @varunagrawal Please review the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,72 @@
-# EvalAI CLI
+# EvalAI-CLI
 
-Official Command Line utility to use EvalAI in your terminal. 
+Official Command Line utility to use EvalAI in your terminal.
 
-This package offers almost all of the full suite of features available on the website within your terminal.
+------------------------------------------------------------------------------------------
+
+<p align="center"><img width="65%" src="docs/static/images/evalai_logo.png" /></p>
```
Comment (deshraj) [inline README.md]: Having this in all the repositories seems a bit redundant. Please remove this.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,72 @@
...
+[![Build Status](https://travis-ci.org/Cloud-CV/evalai-cli.svg?branch=master)](https://travis-ci.org/Cloud-CV/evalai-cli)
+[![Coverage Status](https://coveralls.io/repos/github/Cloud-CV/evalai-cli/badge.svg?branch=master)](https://coveralls.io/github/Cloud-CV/evalai-cli?branch=master)
+
+EvalAI is an open source web application that helps researchers, students and data-scientists to create, collaborate and participate in various AI challenges organized round the globe.
+
+In recent years, it has become increasingly difficult to compare an algorithm solving a given task with other existing approaches. These comparisons suffer from minor differences in algorithm implementation, use of non-standard dataset splits and different evaluation metrics. By providing a central leaderboard and submission interface, we make it easier for researchers to reproduce the results mentioned in the paper and perform reliable & accurate quantitative analysis. By providing swift and robust backends based on map-reduce frameworks that speed up evaluation on the fly, EvalAI aims to make it easier for researchers to reproduce results from technical papers and perform reliable and accurate analyses.
```
Comment (deshraj) [inline README.md]: Please remove this. We don't need to explain it everywhere.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,72 @@
...
+
+<p align="center"><img width="65%" src="docs/static/images/kaggle_comparison.png" /></p>
```
Comment (deshraj) [inline README.md]: Remove this and all the text below this in this section. Seems a lot redundant.

**Comment by RishabhJain2018** [comment]
@deshraj Sure! I will check and add the other details later. Currently, I am merging this PR with badges and contributing guidelines.

**Pull request merged**